src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py

```diff
@@ -52,8 +52,8 @@ def load_command_table(self, _):
     with self.command_group('cosmosdb', cosmosdb_sdk, client_factory=cf_db_accounts) as g:
         g.show_command('show', 'get')
         g.command('list-keys', 'list_keys', deprecate_info=g.deprecate(redirect='cosmosdb keys list', hide=True))
-        g.command('list-read-only-keys', 'list_read_only_keys', deprecate_info=g.deprecate(redirect='cosmosdb keys list', hide=True))
-        g.command('list-connection-strings', 'list_connection_strings', table_transformer=list_connection_strings_output, deprecate_info=g.deprecate(redirect='cosmosdb keys list', hide=True))
+        g.command('list-read-only-keys', 'list_read_only_keys', deprecate_info=g.deprecate(redirect='cosmosdb keys list --read-only-keys', hide=True))
```

Comment:
cosmosdb keys list --type read-only-keys

Reply:
@sinedied can you take a look at the above comment and fix the typo?

Reply:
Ooops my bad, I pushed the fix!
BTW, did you know that you can directly propose the change during reviews in GH for such things? 😉